setup.cfg

```diff
@@ -7,30 +7,26 @@ summary = Image array manipulation functions
 description = Formerly included in SciPy as scipy.stsci.image
 home-page = http://www.stsci.edu/resources/software_hardware/stsci_python
 classifier =
-	Intended Audience :: Science/Research
-	License :: OSI Approved :: BSD License
-	Operating System :: OS Independent
-	Programming Language :: Python
-	Topic :: Scientific/Engineering :: Astronomy
-	Topic :: Software Development :: Libraries :: Python Modules
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Topic :: Scientific/Engineering :: Astronomy
+    Topic :: Software Development :: Libraries :: Python Modules
 requires-python = >=2.5
 requires-dist =
-	stsci.tools
-	numpy (>=1.5.1)
-        scipy (>=0.14)
+    numpy (>=1.5.1)
```

Comment:
Wow seriously? Numpy 1.16 is out, you know.

Reply:
Ha!  I think just changing this to `numpy` and `scipy` would be the best.  It currently builds with the latest versions of both of those.